Jenkinsfile

```diff
@@ -218,17 +242,24 @@ pipeline {
                 unstash 'CIScripts'
                 script {
                     try {
-                        unstash 'testReport'
+                        unstash 'testReportsWindowsCompute'
+                        unstash 'testReportsCISelfcheck'
                     } catch (Exception err) {
                         echo "No test report to parse"
                     } finally {
                         powershell script: '''./CIScripts/GenerateTestReport.ps1 `
-                            -XmlsDir test_report `
-                            -OutputDir processed_reports'''
+                            -XmlsDir testReportsRaw/WindowsCompute `
+                            -OutputDir TestReports/WindowsCompute'''
 
-                        dir("processed_reports") {
-                            stash name: 'processedTestReport', allowEmpty: true
-                        }
+                        powershell script: '''./CIScripts/GenerateTestReport.ps1 `
+                            -XmlsDir testReportsRaw/CISelfcheck `
+                            -OutputDir TestReports/CISelfcheck'''
+
+                        // Using robocopy to workaround 260 chars path length limitation.
```

Comment:
Could you calrify where is this limitation? This comment doesn't seem to explain that.

Reply:
https://github.com/Juniper/contrail-windows-ci/pull/166